portal-api/api-tests/v1/utilisation-report-service/due-report-dates.api-test.js

```diff
@@ -56,6 +41,7 @@ describe('GET /v1/banks/:bankId/due-report-dates', () => {
       dateUploaded,
       uploadedBy: aPaymentReportOfficer,
       path: 'www.abc.com',
+      status: 'REPORT_NOT_RECEIVED',
```

Comment:
As this file is `.js`, we should probably use the constant (or, if you're feeling up for it, converting the file to typescript would be even better)